src/mackerel-container-agent-definition.ts

```diff
@@ -62,9 +62,10 @@ export class MackerelContainerAgentDefinition extends ContainerDefinition {
     }
 
     super(parent, id, {
+      memoryLimitMiB: 128,
```

Comment:
This is necessary only when ECS EC2, but I added this to simplify the code.  the limit value 128 is taken from official doc ref: https://mackerel.io/ja/docs/entry/howto/install-agent/container/ecsawsvpc / https://mackerel.io/ja/docs/entry/howto/install-agent/container/ecs